build.gradle

```diff
@@ -199,22 +201,22 @@ test {
     beforeTest { descriptor ->
         logger.lifecycle("Running test: " + descriptor)
     }
+
+    maxParallelForks parallelism.toInteger()
 }
 
 test.dependsOn(checkSettings)
 
 task testPython(type: Exec, dependsOn: shadowJar) {
-     commandLine 'python3',
-             '-m',
-             'nose',
-             '-c',
-             'python/hail/setup.cfg',
-             'python/hail/tests'
-     environment SPARK_HOME: sparkHome
-     environment PYTHONPATH: '' + projectDir + '/python:' + sparkHome + '/python:' + sparkHome + '/python/lib/py4j-' + py4jVersion + '-src.zip'
-     environment PYSPARK_SUBMIT_ARGS: '--conf spark.driver.extraClassPath=' + projectDir + '/build/libs/hail-all-spark.jar --conf spark.executor.extraClassPath=' + projectDir + '/build/libs/hail-all-spark.jar pyspark-shell'
-     environment PYSPARK_PYTHON: 'python3'
-     environment NOSE_NOLOGCAPTURE: '1'
+    commandLine 'pytest',
```

Comment:
I think I figured why pytest was faster than nose -- we aren't running pytest with coverage enabled

Reply:
I always get error messages abt coverage not being available. I think this was on the cloud too. I can double check.